src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlInternalConnectionX.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NET8_0_OR_GREATER
```

Comment:
Any reason why this class is limited to .NET 8?

Reply:
I'm going to remove this class' inheritance of DbConnection and rename it SqlConnector. This will bring it in line with the Npgsql naming convention. I agree that this class does not need to be limited to .NET 8+.